velox/functions/prestosql/window/tests/NtileTest.cpp

```diff
@@ -78,7 +78,7 @@ TEST_F(NtileTest, singlePartitionWithSortOrders) {
 
 TEST_F(NtileTest, multiInput) {
   NtileTest::testWindowFunction(
-      {makeSinglePartitionVector(75), makeSinglePartitionVector(50)},
+      {makeSinglePartitionVector(100), makeSinglePartitionVector(75)},
```

Comment:
Why is this change?

Reply:
I was trying to manipulate the number of rows to reproduce the problem. Hope that helps.